data/autoyast_sle12/autoinst_sles12sp5_media_minimal_x86.xml

```diff
@@ -0,0 +1,1661 @@
+<?xml version="1.0"?>
```

Comment:
It is too large profile, normally we have the final profile less then 500 lines. :) So need compact.

Reply:
compact version done